content/articles/letsencrypt.markdown

```diff
@@ -0,0 +1,81 @@
+---
+title: Let's Encrypt and DNSimple
+excerpt:
+categories:
+- SSL Certificates
+- Integrations
+---
+
+# Let's Encrypt and DNSimple
+
+<note>
+This article describes a feature in private beta. Please contact support to join the beta and get access to the feature.
+</note>
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+[Let's Encrypt](https://letsencrypt.org/) is a new [certificate authority](/articles/what-is-certificate-authority) that joined the scenes in late 2015, and become an official member of the CA/B forum in 2016.
```

Comment:
Did Let's Encrypt already become an official member of the CA/B forum? If so, this should be "...became an official member..."